webcompat/issues.py

```diff
@@ -56,15 +81,9 @@ def moderation_template(choice='ongoing'):
 
     The default is 'ongoing' even with unknown keywords.
     """
-    if choice == 'rejected':
-        title = REJECTED_TITLE
-        body = REJECTED_BODY
-    elif choice == 'invalid':
-        title = ''
-        body = INVALID_BODY
-    elif choice == 'incomplete':
-        title = ''
-        body = INCOMPLETE_BODY
+    if choice in TEMPLATE_CONFIG:
+        title = TEMPLATE_CONFIG[choice].get('title')
+        body = TEMPLATE_CONFIG[choice].get('body')
     else:
         title = ONGOING_TITLE
         body = ONGOING_BODY
```

Comment:
```suggestion
    title = TEMPLATE_CONFIG[choice].get('title', 'ERROR_TITLE')
    body = TEMPLATE_CONFIG[choice].get('body', 'ERROR_BODY')
```

if everything is in `TEMPLATE_CONFIG`